html2asketch/document.js

```diff
@@ -8,18 +8,20 @@ function pageToPageReference(page) {
   };
 }
 
-function textStyleToSharedStyle(textLayer) {
+function textStyleToSharedStyle(textLayer, documentId, i) {
   return {
     '_class': 'sharedStyle',
-    'do_objectID': generateID(),
+    'do_objectID': `${documentId}/textStyles-${i}`,
     name: textLayer._name,
     'style': textLayer._style.toJSON()
   };
 }
 
+const defaultOptions = {id: 'UI KIT'};
+
 class Document {
-  constructor() {
-    this._objectID = generateID();
+  constructor({id} = defaultOptions) {
+    this._objectID = id || defaultOptions.id;
```

Comment:
If we leave out the objectID entirely here will Sketch use the existing ID of the document that the sketch2asketch plugin is run on? If so, I think that would be the best solution. I found it surprising to inject these asketch files into an existing document only to have them change the identity of the document. Your changes here might make that even worse because by default the plugin would assign the same identity to every document, which might wreak havoc on things like InVision, Sketch libraries, and other Sketch-related tools which use those identifiers.

Reply:
i dont follow. here is how i see this to work (correct me if im wrong somewhere):

Reply:
Before: `objectId` was always random and you have the problem you described.

Reply:
From what I can tell by looking at the source for `asketch2sketch`, it doesn't apply the ID from `document.asketch.json` at all. Instead, it simply uses the currently active document, including its existing ID.

Reply:
That works for me then. Maybe it would reduce confusion to just strip out the objectID here.

Reply:
so, what do we do? /cc @kdzwinel

Reply:
my idea is to have deterministic ids for documents controlled by tools which use this lib and then problem should be eliminated alltogether

Reply:
Since it's an internal property and it's not being used right now - let's kill it to avoid the confusion.